### DISCUSSION TITLE
How to ignore refresh disconnection

### DISCUSSION THREAD

**Discussion by martenzo7** [miguelgrinberg/Flask-SocketIO#2032] [Q&A] [answered]
Hi, I'm trying to make a 'Online members' Database powered by Socket-io. I have a problem when user refresh the page since it's disconnect, I hope there is a way to ignore only the refresh but not when user close the page.

**Comment by miguelgrinberg** [comment, marked as answer]
The browser disconnects when the page refreshes, you cannot keep a connection through a page refresh.

**Comment by martenzo7** [comment]
ok, thanks.
